### DISCUSSION TITLE
A login button for Frigate+ / Better sign up / sign in UX

### DISCUSSION THREAD

**Discussion by Mincka** [blakeblackshear/frigate#8230] [Ideas]
If you already have an account, it's unclear how to connect when you follow this path:
1. https://frigate.video/
2. https://frigate.video/plus/

You are only presented two "Create an account" buttons and only behind them, on page https://plus.frigate.video/ you can "sign in", which is almost the "opposite" issue because people may believe they already need to have an account at that point.

Then, on the page https://auth.frigate.video/, people need to understand that "Sign in" may also mean "Sign up" for Google.
For the email registration, the "Sign up" link is a small link below the form, not clearly visible.

Also, you cannot bookmark directly https://auth.frigate.video/ because the authentication flow does not start, it's a blank page. The intermediary page https://plus.frigate.video/ seems like redundant because we are proposed almost the same actions.

**Comment by blakeblackshear** [comment]
Some of this is a limitation of relying on cognito's hosted UI for auth, but this is good feedback. Thanks.

**Reply by Mincka** [reply]
I can understand that. In that case, just a minor wording change for the buttons with "Sign in or Create an account" on the page https://frigate.video/plus/ would be enough for already registered users. :)
